src/intl/locale.js

```diff
@@ -1,15 +1,446 @@
-export function prioritizeLocales(def, availableLangs, requested) {
-  const supportedLocales = new Set();
-  for (const lang of requested) {
-    if (availableLangs.has(lang)) {
-      supportedLocales.add(lang);
+/* eslint-disable */
```

Comment:
There were over 200 linting errors related to the use of `var`, double quotes and others.  I just disabled linting this file to save some work.